coins/monero/src/wallet/send/mod.rs

```diff
@@ -787,7 +787,7 @@ impl SignableTransaction {
       fee -= output.commitment.amount;
       tx_outputs.push(Output {
         amount: None,
-        key: output.dest.compress(),
+        key: output.dest.compress().to_bytes(),
```

Comment:
Why did you remove the typing from CompressedEdwardsY to [u8; 32]?

Reply:
Reverted